kv/bucket.go

```diff
@@ -182,7 +182,6 @@ func (s *Service) findBucketByName(ctx context.Context, tx Tx, orgID influxdb.ID
 		switch n {
 		case influxdb.TasksSystemBucketName:
 			return &influxdb.Bucket{
-				ID:              influxdb.TasksSystemBucketID,
 				Type:            influxdb.BucketTypeSystem,
 				Name:            influxdb.TasksSystemBucketName,
 				RetentionPeriod: influxdb.TasksSystemBucketRetention,
```

Comment:
This entire switch statement can be deleted and the `if IsNotFound(err)` statement only needs to return the `default` case:

```go
return nil, &influxdb.Error{
    Code: influxdb.ENotFound,
    Msg:  fmt.Sprintf("bucket %q not found", n),
}
```